src/components/v2/headers/ChartHeader.component.tsx

```diff
@@ -102,17 +103,18 @@ function ChartHeaderComponent() {
                 </li>
                 <li className="tab-list__tab">
                     <NavLink
-                        activeClassName="active"
+                        activeClassName="active fcb-5"
                         to={`${match.url}/${URLS.APP_VALUES}`}
-                        className="tab-list__tab-link"
+                        className="tab-list__tab-link flex"
                         onClick={(event) => {
                             ReactGA.event({
                                 category: 'App',
                                 action: 'Values Clicked',
                             })
                         }}
                     >
-                        Values
+                        <Settings className={`tab-list__icon icon-dim-16 fcn-7 mr-4`} />
```

Comment:
We can pass the value directly as string. no need to interpolate

Reply:
Done